crates/wash-lib/src/build.rs

```diff
@@ -180,6 +180,8 @@ fn sign_actor_wasm(
         metadata: ActorMetadata {
             name: common_config.name.clone(),
             ver: Some(common_config.version.to_string()),
+            //NOTE(Ahmed Tadde): sign_file requires a revision, using 1 as default
+            rev: Some(1),
```

Comment:
I think what we should do here is use `Some(0)`, as `0` is treated in wascap by grabbing the current timestamp when signing. Basically if a revision isn't supplied or supplied as 0, then we'll default to a monotonically (mostly) increasing number by default. 
Can @connorsmith256 or @autodidaddict check me on this logic?

Reply:
Correct. If we use a `1` here we'll break developer workflows by blocking updates with no version change.

Reply:
Thanks for double checking me, yeah so I think we should set this to `Some(0)`